### PR TITLE
Scrapper, Ekaterina Nushkina - 22FPL2

### PR DESCRIPTION
**Pull request by eanushkina**

### CODE REVIEW

**Comment by dmitry-uraev** [comment]
good progress,

do not forget to fix imports order in your code 
+
do not forget to implement HTMLParser

**Comment by eanushkina** [comment]
I don't understand how to fix imports order

File: lab_5_scrapper/scrapper.py [hunk 1]
```diff
@@ -2,9 +2,49 @@
...
 from typing import Pattern, Union
 
+import requests
+from bs4 import BeautifulSoup
+
+from core_utils import constants
+from core_utils.article.article import Article
+from core_utils.article.io import to_meta, to_raw
+from core_utils.config_dto import ConfigDTO
+
```
Comment (dmitry-uraev) [inline lab_5_scrapper/scrapper.py:20]: See import style checks log output:

lab_5_scrapper/scrapper.py:21:1: I003 isort expected 1 blank line in imports, found 0

-> add one blank line here

**Comment by dmitry-uraev** [comment]
You have missed deadline and score 4. I suggest you should increase your mark in order to get appropriate mark after lab handover next time.

**Pull request closed without merging**